### ISSUE TITLE
Gui is cutting off tooltips that are too large

### ISSUE THREAD

**Issue by aofengen** [TestRunnerSRL/OoT-Randomizer#1319]
![image](https://user-images.githubusercontent.com/30729364/119576153-31944500-bd86-11eb-9051-6169d24fe971.png)

Hint distro tooltip is too large and is being cut off at the top of the GUI. Either the tooltip needs to extend farther down or the tooltip needs to be able to be scrolled to fix this problem

**Comment by ETR-BTF** [comment]
Can't the tooltip be extended horizontally?

**Comment by aofengen** [comment]
I think that would only push the problem further down the road until happens again - if the tooltips can be scrollable, it would solve everything, to be honest

**Comment by Malkierian** [comment]
A little disappointed this hasn't been dealt with yet. It *should* be a pretty simple fix (though I say "should" because I don't know the underlying code, so I can't say for sure). But, I wanted to also add a dimension to this issue. If there's enough horizontal screen space for the tooltip to fit underneath the GUI element it's referring to, when it comes to dropdowns, the tooltips display below the dropdown box, and only display when the dropdown is open, and thus the dropdown options actually cover the top of the tooltip:
![image](https://user-images.githubusercontent.com/774771/152698472-29d50832-6d85-4efa-9f56-46dd3d1a456e.png)

**Comment by aofengen** [comment]
May rebase and look at this again later

**Comment by fenhl** [comment]
I don't think this issue should be closed, it's still relevant in current Dev.

**Comment by aofengen** [comment]
You're right that it's still an issue - pretty sure I deleted the branch a while back, though. If so, will just rebase and recreate a new PR - if I still have the branch, will rebase and reopen this one @fenhl

**Comment by fenhl** [comment]
This is an issue, not a PR.

**Comment by aofengen** [comment]
Ok, I need to stop doing things this early in the morning. Thought this was one of my PRs ._.
